actionview/lib/action_view/record_identifier.rb

```diff
@@ -60,7 +60,7 @@ module RecordIdentifier
     include ModelNaming
 
     JOIN = "_"
-    NEW = "new"
+    NEW = ["new"]
```

Comment:
While testing locally I could revert this back to a plain string:
```suggestion
    NEW = "new"
```